docs/developer/releasing.md

```diff
@@ -31,6 +31,14 @@ This will automatically create a PR in [k8s.io](https://github.com/kubernetes/k8
 1. Review the draft release on GitHub. Pay close attention to the `## :question: Sort these by hand` section, as it contains items that need to be manually sorted.
 1. Publish the release
 
+## Update Netlify for Cluster API book
```

Comment:
should we make a note that only Netlify admins have permissions so the task is really ask an admin to do the below things?

Reply:
done!